lib/teiserver_web/templates/admin/user/tab_overview.html.heex

```diff
@@ -107,7 +107,15 @@
   <%= central_component("menu_card",
     icon: "fa-solid #{Teiserver.Moderation.ReportLib.icon()}",
     name: "moderation",
-    text: "Moderation",
+    text: "Reports",
+    icon_class: "fa-solid",
+    link: ~p"/moderation/report/user/#{@user.id}"
```

Comment:
You don't need to specify `user.id`, there's this [protocol param](https://hexdocs.pm/phoenix/Phoenix.Param.html) that will automatically take care of that for you. `~p"/moderation/report/user/#{@user}"` would do.

Reply:
Didn't know, that's pretty cool, thank you!